aspnetcore/mvc/views/overview.md

```diff
@@ -1,228 +1,332 @@
+}
+```
 
-<!-- literal_block {"ids": [], "linenos": false, "xml:space": "preserve", "language": "csharp", "highlight_args": {"hl_lines": [13]}} -->
+There are no restrictions on the model types that you can provide to a view. We recommend using **P**lain **O**ld **C**LR **O**bject (POCO) viewmodels with little or no behavior (methods) defined. Usually, viewmodel classes are either stored in the *Models* folder or a separate *ViewModels* folder at the root of the app. The *Address* viewmodel used in the example above is a POCO viewmodel stored in a file named *Address.cs*:
 
 ```csharp
 namespace WebApplication1.ViewModels
-   {
-       public class Address
-       {
-           public string Name { get; set; }
-           public string Street { get; set; }
-           public string City { get; set; }
-           public string State { get; set; }
-           public string PostalCode { get; set; }
-       }
-   }
-   ```
+{
+    public class Address
+    {
+        public string Name { get; set; }
+        public string Street { get; set; }
+        public string City { get; set; }
+        public string State { get; set; }
+        public string PostalCode { get; set; }
+    }
+}
+```
 
 > [!NOTE]
-> Nothing prevents you from using the same classes as your business model types and your display model types. However, keeping them separate allows your views to vary independently from your domain or persistence model, and can offer some security benefits as well (for models that users will send to the app using [model binding](../models/model-binding.md)).
+> Nothing prevents you from using the same classes for both your viewmodel types and your business model types. However, using separate models allows your views to vary independently from the business logic and data access parts of your app. Separation of models and viewmodels also offers security benefits when models use [model binding](xref:mvc/models/model-binding) and [validation](xref:mvc/models/validation) for data sent to the app by the user.
+
+### Weakly-typed data (ViewData and ViewBag)
+
+In addition to strongly-typed views, views have access to a *weakly-typed* (also called *loosely-typed*) collection of data. Unlike strong types, *weak types* (or *loose types*) means that you don't explicitly declare the type of data you're using. You can use the collection of weakly-typed data for passing small amounts of data in and out of controllers and views.
+
+| Passing data between a ...                        | Example                                                                        |
+| ------------------------------------------------- | ------------------------------------------------------------------------------ |
+| Controller and a view                             | Populating a dropdown list with data.                                          |
+| View and a [layout view](xref:mvc/views/layout)   | Setting the **\<title>** element content in the layout view from a view file.  |
```

Comment:
**\<title>** --> `<title>`

Reply:
This is one I may have forgotten/mis-read, but I thought the style manual called for bold elements. IIRC it's on that summary styling chart.

Reply:
Yeah ... you know the chart I mean? ... it's a big summary formatting chart. I think it's at the top of that chart. I asked about this once. It was several months (or a year!) ago, but I don't recall the outcome of the conversation. I do recall the entry in the chart tho.

Reply:
@guardrex I see the rule you're talking about now. You're correct. Let's stick to bold.